subprojects/dependency-management/src/main/java/org/gradle/api/internal/filestore/ivy/ArtifactIdentifierFileStore.java

```diff
@@ -32,9 +35,10 @@ public String determineGroup(ModuleComponentArtifactIdentifier artifactId) {
 
         @Override
         public int getDepth() {
-            return 3;
+            return FILE_TREE_DEPTH_TO_TRACK_AND_CLEANUP - 1;
```

Comment:
⭕️ This may read clearer if it was also a constant and the cleanup constant was derived from it.

Reply:
Agreed, I will change it.